This patch will remove the border which is needed otherwise there
is an empty one-pixel border around windows even when gaps are
eliminated.

--- config.h	2021-11-29 17:12:31.120209141 -0500
+++ config.h	2021-11-29 17:12:31.040206941 -0500
@@ -2,7 +2,7 @@
 #include <X11/XF86keysym.h> /* for volume and brightness keys */
 
 /* appearance */
-static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const unsigned int borderpx  = 0;        /* border pixel of windows */
 static const unsigned int gappx     = 5;        /* gaps between windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
